src/search/query.rs

```diff
@@ -31,6 +32,14 @@ impl Query {
         }
     }
 
+    /// *Adds* an author to the query.
+    ///
+    /// Result is `author:username`.
+    pub fn author(mut self, author: &str) -> Self {
```

Comment:
Similarly to the note I left in #23, could you change the documentation to more closely match the function parameters?

Reply:
👌